### ISSUE TITLE
--volume incorrect volume format "Z"

### ISSUE THREAD

**Issue by QiWang19** [containers/buildah#1685]
**Description**

**Steps to reproduce the issue:**
playing with buildah run --volume command from buildah-run.md
```
buildah run --volume /path/on/host:/path/in/container:ro,z containerID sh
```
if this is an issue, buildah/run.go https://github.com/containers/buildah/blob/c22957b5be1b5463c8be22d7380a08f48155d256/cmd/buildah/run.go#L74 should use flags.StringArrayVarP
**Describe the results you received:**
incorrect volume format "Z", should be host-dir:ctr-dir[:option]

**Describe the results you expected:**
return without error

**Output of `rpm -q buildah` or `apt list buildah`:**

```
buildah-1.8.2-1.gite23314b.fc30.x86_64
```

**Output of `buildah version`:**

```
Version:         1.9.1-dev
Go Version:      go1.12.5
Image Spec:      1.0.0
Runtime Spec:    1.0.0
CNI Spec:        0.4.0
libcni Version:  v0.7.0-rc2
Git Commit:      2c4f388b
Built:           Wed Jun 19 14:51:17 2019
OS/Arch:         linux/amd64

```

**Comment by rhatdan** [comment]
@QiWang19 Please fix.

**Comment by rhatdan** [comment]
@QiWang19 Nice find.  BTW.

**Comment by TomSweeneyRedHat** [comment]
Yeah, especially since she proposed the probable fix, my favorite kind of issue!

**Comment by nalind** [comment]
This looks to have been fixed by #1686, closing.
